### PR TITLE
update: get fresh look + clean

### PR DESCRIPTION
**Pull request by AugustinMauroy**
- add support of dark theme
- use green instead of brown to look like nodejs.org
- Adding padding on `code` to give "fresh" air to the text
- add robots.txt

## Light Theme

Capture d’écran 2024-03-29 à 11 25 23

## Dark Theme

Capture d’écran 2024-03-29 à 11 24 45

## Light house

Capture d’écran 2024-03-29 à 11 33 30

### CODE REVIEW

File: getting-started/index.html [hunk 1]
```diff
@@ -1,6 +1,13 @@
-<!doctype html>
-<meta charset="utf-8">
-<title>Redirect to home page</title>
-<script>window.location="https://www.nodetodo.org/";</script>
```
Comment (Trott) [inline getting-started/index.html:4]: With this `<script>` tag removed, won't it break the automatic redirect?

**Comment by Trott** [comment]
I appreciate these changes, but it would be easier to review and land if this were split into one PR per change (or at least one commit per change): One PR (or commit) that adds the 404 page, one PR (or commit) that does color/branding changes, one PR (or commit) that does a11y improvements, that sort of thing. This way, adding a 404 page doesn't have to be held up on whether or not the redirect page is ready to go, and so on.

**Comment by Trott** [comment]
I especially have to ponder the branding change. I don't want this to look like Node.js's site because it's not Node.js. It's unofficial. It's a little thing I maintain. So taking Node.js colors and branding and using those might not be what I want to do. (But also: Maybe it is.) So that part especially would be helpful if it could be split into it's on PR (or at least it's on commit that can be omitted if I so choose).

**Comment by Trott** [comment]
Thanks for these updates! People (including me!) very rarely think to try to improve NodeTodo.

File: getting-started/index.html (hunk 1 above)
Comment (AugustinMauroy) [reply getting-started/index.html:4]: I'll re-add fail of my part 😄

**Comment by AugustinMauroy** [comment]
>  I don't want this to look like Node.js's site because it's not Node.js. It's unofficial

I agree with you. But I'm thinking of cyan or blue because brown isn't very cheerful.

> I appreciate these changes, but it would be easier to review and land if this were split into one PR per change (or at least one commit per change): One PR (or commit) that adds the 404 page, one PR (or commit) that does color/branding changes, one PR (or commit) that does a11y improvements, that sort of thing. This way, adding a 404 page doesn't have to be held up on whether or not the redirect page is ready to go, and so on.

So I close this pr and I create one for 404, one for "re-brand" an other for robots.txt ... ????

**Comment by Trott** [comment]
> So I close this pr and I create one for 404, one for "re-brand" an other for robots.txt ... ????

If it's not a lot of tedious work, yes, that would be great. If it is a lot of tedious work, then this will work well enough.

**Comment by AugustinMauroy** [comment]
> If it's not a lot of tedious work, yes, that would be great. If it is a lot of tedious work, then this will work well enough

No worries, 3 minutes is all it takes. I'll do it tomorrow morning (European time).

**Pull request closed without merging**